### ISSUE TITLE
Generate tests button not showing when using export default class in typescript

### ISSUE THREAD

**Issue by david-robinson-practiceweb** [Codium-ai/codiumai-jetbrains-release#49]
Was just having a try of codiumAI with version  0.5.14 in PHPStorm 2023.1.2
When. declaring a class directly as part of the export default the generate tests button wasn't appearing
```
export default class basic {
}
```
It does appear successfully  if the class is declared and then exported as default
```
class basic {
 
}
export default basic;
```
it does  appear successfully on exporting  non default classes as part of the export declaration.
```
export class basic2 {

}
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/eaQWzh3i) by [Unito](https://www.unito.io)

**Comment by ArtsiomCh** [comment]
Many Thanks for reporting!
Indeed we missed that case. Fixed in upcoming 0.5.15 (should be available beginning of coming week)

**Comment by david-robinson-practiceweb** [comment]
Thanks for the fix - just got the update to 0.5.15 and the generation prompt is showing now.
